.github/workflows/build-and-test.yml

```diff
@@ -42,7 +42,10 @@ jobs:
             matrix:
                 python-version: [3.6, 3.7]
                 python-architecture: [x86, x64]
-                os: [windows-latest]
+                os: [windows-latest, macos-latest]
```

Comment:
I added to the os matrix here, rather than making a separate job for mac tests. That lets us re-use the step of checking out test data, but then it also requires a bunch of conditionals checking the os is subsequent steps. I'm not sure if this was a good idea or not.

Reply:
I think the ideal case here would be that we're using the same setup and test configuration across all of our target OSes, and I think that's a good thing for us to strive for.  To that end, defining the OS as part of the build matrix makes very good sense to me.